bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/hbasesnapshots/EndToEndIT.java

```diff
@@ -178,27 +148,19 @@ public void teardown() throws IOException {
       this.gcsUtil.remove(pathStrs);
     }
 
+    connection.getAdmin().deleteTable(TableName.valueOf(tableId));
     connection.close();
-
-    // delete test table
-    BigtableConfiguration.connect(projectId, instanceId)
-        .getAdmin()
-        .deleteTable(TableName.valueOf(tableId));
   }
 
   private SyncTableOptions createSyncTableOptions() {
     DataflowPipelineOptions syncTableOpts =
         PipelineOptionsFactory.as(DataflowPipelineOptions.class);
-    syncTableOpts.setRunner(DataflowRunner.class);
-    syncTableOpts.setGcpTempLocation(dataflowStagingLocation);
```

Comment:
i don't think properties.set is setting this option either?

Reply:
I dont think its required...I removed unnecessary opts